temba/msgs/models.py

```diff
@@ -140,7 +142,7 @@ def create(
             text = {base_language: text}
 
         assert groups or contacts or contact_ids or urns, "can't create broadcast without recipients"
-        assert base_language in text, "base_language doesn't exist in text translations"
+        # assert base_language in text, "base_language doesn't exist in text translations"
```

Comment:
Shouldn't base_language be still asserted in default broadcasts?

Reply:
You're right, fixed!